aio/aio-proxy/aio_proxy/response/build_response.py

```diff
@@ -5,31 +5,33 @@
 from aio_proxy.decorators.http_exception import http_exception_handler
 from aio_proxy.response.format_response import format_response
 from aio_proxy.response.format_search_results import format_search_results
-from aio_proxy.search.index import ElasticsearchSireneIndex
+from aio_proxy.search.es_search_runner import ElasticSearchRunner
 from aiohttp import web
 from sentry_sdk import capture_exception, push_scope
 
 
 @http_exception_handler
```

Comment:
Might be off topic for now but I would merge the `excpet` at the bottom of the file into this handler

Reply:
Also coming soon ;)